.drone.yml

```diff
@@ -0,0 +1,116 @@
+workspace:
+  base: /go
+  path: src/github.com/src-d/code-annotation
+
+branches: [master, staging, release/*]
+
+pipeline:
+
+  clone:
+    image: plugins/git
+    debug: true
+
+
+  # build on pull requests
+
+  build:
+    image: golang:1.8-alpine3.6
+    environment:
+      - REACT_APP_SERVER_URL="//code-annotation.sourced.tech"
+    commands:
+      - apk --update upgrade
+      - apk add --no-cache make git curl ca-certificates bash build-base libxml2-dev protobuf nodejs=6.10.3-r1 nodejs-npm
+      - npm install -g yarn
+      - make build
+      - make packages
+    debug: true
+    when:
+      event: [pull_request]
+
+
+  # deployment to staging environment
+
+  build_stg:
+    image: golang:1.8-alpine3.6
+    environment:
+      - REACT_APP_SERVER_URL="//code-annotation.srcd.run"
```

Comment:
expected url would be `//code-annotation-staging.srcd.run`
same pattern than for: `talks-staging.srcd.run`, `blog-staging.srcd.run`, and `landing-staging.srcd.run`
https://github.com/src-d/guide/blob/master/engineering/continuous-delivery.md